### DISCUSSION TITLE
Interrupt and Resume AVCaptureSession

### DISCUSSION THREAD

**Discussion by Parvinderjit** [HaishinKit/HaishinKit.swift#1343] [Q&A]
When we get the any audio interrupt the system suspends the capture session and camera got freeze. But if we put the app in the background and after some time we open the app again it won't resume the capture session. It still have that frozen camera preview. Earlier I was starting the session from the mixer.session but now the mixer property is internal to the module. So not able to access the session property of mixer. Can anyone let me know how can we handle that situation?

**Comment by shogo4405** [comment]
I was wondering if you could handle this using the delegate method here.
https://github.com/shogo4405/HaishinKit.swift/blob/main/Sources/Net/NetStream.swift#L22-L27
